.github/workflows/docker-package.yml

```diff
@@ -20,15 +20,11 @@ jobs:
   build-and-publish:
     name: Build and Publish Docker Image
     runs-on: ubuntu-20.04
-    # For Pull Requests, only runs from `docker`, `feat`, `fix`, `patch`, or `dependabot`
+    # For Pull Requests, only runs from `docker`
```

Comment:
Only because I don't know these details: this doesn't affect gitpod devcontainers, does it?

Reply:
That's a good question... I don't think so? Hopefully the pod is based on the main branch image

Reply:
Cc @MatthewFlamm just to be safe.

Reply:
This build action does not involve any builds for gitpod or devcontainer which are currently built from supported base images from the respective providers and built using their machinery.  I'm not actually sure what these build actions are used for, so I have no real comment on this PR.